src/App.vue

```diff
@@ -718,23 +713,37 @@ export default class App extends Mixins(
 
     // fetch draft filing
     // NB: will throw if API error
-    let draftFiling = await this.fetchDraftIA()
+    let draftFiling = await this.fetchDraftApplication()
```

Comment:
You might need to catch a thrown error here. I see that line 725 checks for a null response, but you might also see other errors, which you should handle gracefully.

Reply:
Sorry i forgot to update here.
An error in the request or if the filing is somehow fails to build/null it will be caught in our `fetchData` in our `catch` block starting on line 623 of `app`

Reply:
Oh, right, that's a try-catch block that wraps this function.